azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -1153,6 +1153,32 @@ def connection_string(self, device_connection_string):
     def sas_token_string(self, device_sas_token_string):
         return device_sas_token_string
 
+    @pytest.fixture
+    def fake_assigned_hub(self):
+        return "DumbledoreArmy.azure-devices.net"
+
+    @pytest.fixture
+    def fake_device_id(self):
+        return "elder_wand"
+
+    @pytest.fixture
+    def symmetric_key(self):
+        return "Zm9vYmFy"
+
+    @pytest.fixture
+    def registration_result(self, fake_assigned_hub, fake_device_id):
+        class FakeRegistrationState(object):
```

Comment:
>FakeRegistrationState [](start = 14, length = 21)

You have RegistrationState and RegistrationResult models in the client.  Why not use those?

Reply:
Technically I only need the device id and assigned hub from those. But yes i can change it.

---
In reply to: [355038779](https://github.com/Azure/azure-iot-sdk-python/pull/407#discussion_r355038779) [](ancestors = 355038779)